src/launch/launch.html

```diff
@@ -1,5 +1,31 @@
 <template>
+  <require from="./launch-box.html"></require>
   <div class="page animated-page au-animate launch">
-    Launch
+    <div class="banner">
+      <div class="banner1">
+        <span> Apply to host </span>
+        <span style="padding-left: 1vw"> your Launch </span>
+      </div>
+      <div class="banner2">
+        <span>
+          <p>Prime Launch empowers and creators to host and fair and</p>
```

Comment:
I believe it is best not to use 'p' because it may be implemented differently o different browsers.  better to stick with div and a class